TokenService.js

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2018-2020 Digital Bazaar, Inc. All rights reserved.
  */
 
-import axios from 'axios';
+import {httpClient} from '@digitalbazaar/http-client';
 import bcrypt from 'bcryptjs';
```

Comment:
Alphabetize imports.

```suggestion
import bcrypt from 'bcryptjs';
import {httpClient} from '@digitalbazaar/http-client';
```